src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/InterproceduralAnalysisConfiguration.cs

```diff
@@ -43,39 +43,40 @@ public struct InterproceduralAnalysisConfiguration : IEquatable<InterproceduralA
             ControlFlowGraph cfg,
             Compilation compilation,
             InterproceduralAnalysisKind defaultInterproceduralAnalysisKind,
+#pragma warning disable CA1801 // Review unused parameters - Public API
```

Comment:
Its fine to make a breaking public change for this package. I'd remove the unused parameter.